### ISSUE TITLE
Get peripheral when call setBlockOnDidUpdateNotificationStateForCharacteristicAtChannel:

### ISSUE THREAD

**Issue by anhhtbk** [coolnameismy/BabyBluetooth#142]
I connect to multiple peripheral, so I want to know what peripheral call updateNotification.
Can I do it? and How?
Thanks!

**Comment by AlainHsu** [comment]
I don't think it can be done unless the peripheral object has been added in the return.

**Comment by yinanwang1** [comment]
Yes, I think so. To add the peripheral parameter in the block is a good way.
